### PR TITLE
incubator/fluentd-cloudwatch Added annotations for the service account

### PR DESCRIPTION
**Pull request by stijndehaes**
Signed-off-by: Stijn De Haes <stijndehaes@gmail.com>

#### Is this a new chart
No
#### What this PR does / why we need it:

It adds service account annotations, so it is possible to use the EKS iam integrations

#### Which issue this PR fixes
None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/fluentd-cloudwatch/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -9,3 +9,6 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 {{- end }}
+{{- if .Values.rbac.serviceAccountAnnotations }}
+  annotations: {{ toYaml .Values.rbac.serviceAccountAnnotations | nindent 4 }}
```
Comment (davidkarlsen) [inline incubator/fluentd-cloudwatch/templates/serviceaccount.yaml]: shouldn't this be on a newline?

Comment (stijndehaes) [reply incubator/fluentd-cloudwatch/templates/serviceaccount.yaml]: I thought it weird too. But I used the same approach as with cluster autoscaler. Seems to work fine there:
https://github.com/helm/charts/blob/master/stable/cluster-autoscaler/templates/serviceaccount.yaml#L10

Comment (stijndehaes) [reply incubator/fluentd-cloudwatch/templates/serviceaccount.yaml]: To be sure I put it on a newline

**Comment by stijndehaes** [comment]
@davidkarlsen Would you be ok with doing another review?

**Comment by zanhsieh** [comment]
> @davidkarlsen Would you be ok with doing another review?

@stijndehaes 
Did you click the circle button to request @davidkarlsen review again, like below?
image

**Comment by stijndehaes** [comment]
> > @davidkarlsen Would you be ok with doing another review?
> 
> @stijndehaes
> Did you click the circle button to request @davidkarlsen review again, like below?
> 
> image
> 
> That button only show between the reviewer asking the contributor to change. No other people could see it or click it. So unfortunately none of people in this PR could help directly ...

@zanhsieh Yes I did :) I just didn't hear from him for 10 days. So I thought I would send him a friendly reminder. Should I not have? Because if that is a problem I won't anymore :)

**Comment by zanhsieh** [comment]
@stijndehaes 
How about shout out in kubernetes slack #charts channel for other helm member attention?

**Pull request merged**